CMakeLists.txt

```diff
@@ -511,14 +522,6 @@ else()
 	message(FATAL_ERROR "Invalid target platform")
 endif()
 
-if(N3DS)
```

Comment:
This must stay this way for libretro. These platform specific codepaths must always be used otherwise Player doesn't compile.

Reply:
Ah okay, I think I remember talking about this with you in an earlier pull request. Will revert.